lms/events/__init__.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+
+from pyramid.events import subscriber
+from pyramid.request import Request
+
+from lms.models import Event, EventType, LtiLaunches
+from lms.services import EventService, LTIRoleService
+
```

Comment:
As discussed before, yeah I think it makes sense to use pyramid events for this.

We should keep events tied to things that don't seem fundamental to the linear executing of the code.

It's a fine line but I think, emitting and event and storing some metadata in the DB is fine, but doing the same thing for for example storing the assignment would be too much.